tekton/workshop/content/lab06.md

```diff
@@ -199,66 +205,76 @@ So, while the ClusterTask seemed like a decent first pass at how to build our ap
-However, the examples below really lean into providing some "local resources" (e.g. such as a git repository clone) to the container, which would be more complicated in OpenShift (e.g. setting up a Persistent Volume with the checkout, providing that PV to the container, etc) which would cancel the power and simplicity of the "local development" examples below. 
+OK, now we're back to the drawing board - we couldn't use the `ClusterTask` out of the box with our existing knowledge, but let's see what we can learn from it.
+
+First off, let's talk about the importance of feedback loops. As engineers, we always want to have a short and tight feedback loop. The tighter the feedback loop, the faster we could experiment, and then the faster we could learn. From experience, the tightest feedback loop happens when engineers can play with things locally on their workstations. How could we do that with Tekton? Luckily, since Tekton is based entirely on running containers, we have a way!
 
----
+Reviewing the content of the `maven` `ClusterTask`, we can see exactly what container image the `ClusterTask` uses to execute our maven goals:
 
-OK, now we're back to the drawing board - we couldn't use the ClusterTask out of the box with our existing knowledge, but let's see what we can learn from it.
+```execute
+tkn ct describe maven -o jsonpath="{.spec.steps[?(@.name=='mvn-goals')].image}" && echo
 
-First off, let's talk about the importance of feedback loops. As engineers, we always want to have a short and tight feedback loop. The tighter the feedback loop, the faster we could experiment, and then the faster we could learn. From experience, the tightest feedback loop happens when engineers can play with things locally on their workstations. How could we do that with Tekton ? Luckily, since Tekton is based entirely on running containers, we have a way !
+```
 
-Re-reviewing the content of the `maven` ClusterTask, we can see exactly what container image the ClusterTask uses : `gcr.io/cloud-builders/mvn` . If we're curious, we pull down that image and explore it by using it just like any other container. We will check what version of Maven is in there using `mvn -version`, and then we will `exit`:
-* The `-it` argument to `podman` gives us an "interactive tty/shell", whereas the `--entrypoint` argument allows us to override what will be executed when the container starts (the equivalent podman command is ``)
+If we're curious, we pull down that image and explore it by using it just like any other container. In OpenShift, we can run an individual container within a `Pod` using `oc run`. This is useful for situations like ours, when we need an ephemeral test environment. Execute the command below:
```

Comment:
I like shifting this up to replace the full-local dev. Helps demonstrate the power of the platform for dev right here. :clap: